test/github-auth.js

```diff
@@ -68,6 +68,15 @@ class FakeGithub {
       'integrations.getInstallationRepositories': async() => {
         return this.repositories;
       },
+      'users.getForUser': async ({username}) => {
+        let user = _.find(this.github_users, {username});
+        if (user) {
+          user.id = parseInt(user.id, 10);
```

Comment:
It looks like `user.getForId` returns the value as an integer (https://api.github.com/users/djmitche).  And `setUser` is just a test function, so if you always pass it an integer, there's no need to do anything.